include/opendb/db.h

```diff
@@ -32,6 +32,9 @@
 #ifndef ADS_DB_H
 #define ADS_DB_H
 
+// These goofy looking protection checks save sending the contents of the
```

Comment:
"#pragma once" is generally a better solution to this problem.

Reply:
Feel free to jump in your time machine and explain that to Tom Giacarous (sic).